src/Microsoft.DocAsCode.Common/Path/RelativePath.cs

```diff
@@ -236,6 +241,12 @@ public RelativePath UrlDecode()
             {
                 parts[i] = Uri.UnescapeDataString(_parts[i]);
             }
+
+            if (_parts.Length > 0 && parts[0] == WorkingFolderString)
+            {
+                return new RelativePath(true, _parentDirectoryCount, parts.Skip(1).ToArray());
```

Comment:
This change is in UrlDecode method. Since after decode the path, 7E% may become ~, so we need to skip the first part(~) and set _isFromWorkingFolder to true.